sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero_command_buffer.cpp

```diff
@@ -582,9 +583,130 @@ UR_APIEXPORT ur_result_t UR_APICALL urCommandBufferAppendMembufferCopyRectExp(
                                  hCommandBuffer->Device));
 
   return enqueueCommandBufferMemCopyRectHelper(
-      hCommandBuffer, ZeHandleDst, ZeHandleSrc, srcOrigin, dstOrigin, region,
-      srcRowPitch, dstRowPitch, srcSlicePitch, dstSlicePitch,
-      numSyncPointsInWaitList, pSyncPointWaitList, pSyncPoint);
+      UR_COMMAND_MEM_BUFFER_COPY_RECT, hCommandBuffer, ZeHandleDst, ZeHandleSrc,
+      srcOrigin, dstOrigin, region, srcRowPitch, dstRowPitch, srcSlicePitch,
+      dstSlicePitch, numSyncPointsInWaitList, pSyncPointWaitList, pSyncPoint);
+}
+
+UR_APIEXPORT ur_result_t UR_APICALL urCommandBufferAppendMembufferWriteExp(
+    ur_exp_command_buffer_handle_t hCommandBuffer,
+    ur_mem_handle_t hBuffer, ///< [in] handle of the buffer object
```

Comment:
We don't need the ` ///< [in] handle of the buffer object` comments here, that should be documented in the autogenerated UR header